src/openrct2-ui/interface/Widget.cpp

```diff
@@ -346,9 +346,8 @@ static void widget_text_centred(rct_drawpixelinfo* dpi, rct_window* w, rct_widge
 
     if (widget->type == WWT_BUTTON || widget->type == WWT_TABLE_HEADER)
     {
-        int32_t height = widget->height();
-        if (height >= 10)
-            topLeft.y += std::max<int32_t>(widget->top, widget->top + (height / 2) - 5);
+        if (widget->height() >= 10)
+            topLeft.y += std::max<int32_t>(widget->top, widget->top + (widget->height() / 2) - 5);
```

Comment:
This is not an improvement. Did you plan to use midY here?

Reply:
At the moment, this is just inlining the `height` variable, as it is not needed now this function exists. However, using `midY` would be shorter and may be preferable, yes. I’m not sure it yields the exact same result though. Worth looking into.

Reply:
I think *just* inlining the variable only makes for slightly harder to read code and does not have any advantages. I'd like to see this either reverted or modified in a way that makes it easier to read.

Review note: this also applies to other occurrences in this file. All other files look good to me.